inc/namespace.php

```diff
@@ -35,6 +35,8 @@ function enqueue_block_editor_assets() {
 		'restBase' => esc_url_raw( get_rest_url() ),
 		'postSelectEndpoint' => '/rbmh/v1/post-select',
 	] );
+
+	wp_set_script_translations( 'hm-gb-tools-editor', 'rbmh' );
```

Comment:
`rbmh` is the text domain from the project this was built for, we should change all instances of rbmh to something else. e.g. `hm-gb-tools`?